share/pulse/examples/by-example/PulseTutorial.Loops.fst

```diff
@@ -204,10 +204,10 @@ noextract
  //fib_rec$
 fn rec fib_rec (n:pos) (out:ref (nat & nat))
 requires
-    pts_to out 'v
+    pts_to out #1.0R 'v
```

Comment:
surprising change

Reply:
Yes, and apparently it's no longer necessary! 😄  I've reverted it now.